inventory/host_vars/migration.katuma.org/config.yml

```diff
@@ -0,0 +1,21 @@
+---
+domain: app.katuma.org
+host_id: es-prod
+rails_env: production
+
+admin_email: info@coopdevs.org
+
+mail_domain: katuma.org
+
+certbot_domains:
+  - app.katuma.org
+  - alpha.katuma.org
+  - migration.katuma.org
```

Comment:
Note for future migrations: this `certbot_domains` bit is the whole point of this temporary config file. We list both the current server's domain(s) and the new server's domain, with the current server at the top of the list. Also note the `domain` variable at the top of the file should match the current server and not the new server we're migrating to.

We can then use the `letsencrypt_proxy` playbook on the current server and then provision the new server, and it'll have a certificate that's valid for both domains even though we haven't done the migration yet :ok_hand: